constantine/math/arithmetic/finite_fields_precomp_square_root.nim

```diff
@@ -24,20 +24,19 @@ import
 # The returned value is only meaningful modulo 1<<sqrtParam_BlockSize and is fully reduced, i.e. in [0, 1<<sqrtParam_BlockSize )
 #
 # NOTE: If x is not a root of unity as asserted, the behaviour is undefined.
-func sqrtAlg_NegDlogInSmallDyadicSubgroup*(x: Fp): int =
+func sqrtAlg_NegDlogInSmallDyadicSubgroup(x: Fp): int =
```

Comment:
```suggestion
func sqrtAlg_NegDlogInSmallDyadicSubgroup_vartime(x: Fp): int {.tags:[VarTime], raises: [].} =
```

There might be errors that might ask you to add other effects or mention unlisted exceptions